src/main/resources/application-local.yml

```diff
@@ -35,6 +35,6 @@ no.nav.security.jwt:
   issuers: selvbetjening
   issuer.selvbetjening:
     discoveryurl: http://metadata
-    accepted_audience: aud-localhost
+    acceptedaudience: aud-localhost
```

Comment:
jeg tror kanskje det skal være underscore? 🤔 
https://github.com/navikt/token-support#required-properties-yaml-or-properties

Reply:
Endret på accepted_audience for prod og preprod også.